app/api/product.py

```
@@ -1,167 +1,262 @@
+        active_options_stmt = select(func.count(DBProductOption.id)).where(DBProductOption.is_deleted == False)
+        options_result = await db.execute(active_options_stmt)
+        active_options = options_result.scalar() or 0
+        
+        # 추천 서비스 통계
+        recommendation_stats = recommendation_service.get_recommendation_stats()
 
-@router.post("/embedding/generate")
-async def generate_product_embedding(
+        # 벡터 저장소 통계
+        vector_stats = recommendation_service.vector_store.get_store_stats()
+        indexed_products = vector_stats["index_stats"]["total_vectors"]
```

Ruff (0.11.9)

Local variable `indexed_products` is assigned to but never used

Remove assignment to unused variable `indexed_products`

(F841)